src/lib/data/alignment.js

```diff
@@ -717,19 +717,29 @@ export default class Alignment {
     return alignedText.segments[token.segmentIndex - 1]
   }
 
+  getNextSegmentByToken (token) {
+    const alignedText = this.getAlignedTextByToken(token)
+    return alignedText.segments.length > token.segmentIndex ? alignedText.segments[token.segmentIndex] : false
+  }
+
+  getPrevSegmentByToken (token) {
+    const alignedText = this.getAlignedTextByToken(token)
+    return token.segmentIndex > 1 ? alignedText.segments[token.segmentIndex - 2] : false
+  }
+

```

Comment:
i think returning null rather than false makes more sense when the next/prev token doesn't exist

Reply:
fixed